dango/lending/src/execute.rs

```diff
@@ -150,3 +151,30 @@ pub fn borrow(ctx: MutableCtx, coins: Coins) -> anyhow::Result<Response> {
     // Transfer the coins to the caller
     Ok(Response::new().add_message(Message::transfer(ctx.sender, coins)?))
 }
+
+pub fn repay(ctx: MutableCtx) -> anyhow::Result<Response> {
+    // Ensure all sent coins are whitelisted
+    for coin in &ctx.funds {
+        ensure!(
+            MARKETS.has(ctx.storage, coin.denom),
+            "Invalid denom. Only whitelisted denoms can be repaid."
+        );
+    }
```

Comment:
Note: in practice, we don't need to assert this, because once we have implemented interest rate, we need to update interest rate state here, so if the denom isn't whitelisted, this is will naturally fail.

Reply:
True, but that error would probably be quite confusing I'd assume?